omero/developers/Web/Deployment.txt

```diff
@@ -58,142 +58,7 @@ Debugging on and start the server up:
 Using FastCGI (Unix/Linux)
 --------------------------
 
-Once you have installed a FastCGI capable web server, configuration of
```

Comment:
@manics this doc was created to adress the difference between bin/omero web config nginx and --system. I wouldn't remove it